### DISCUSSION TITLE
Library Size is 111KB not like said " lightweight framework (~20K gzipped)."

### DISCUSSION THREAD

**Discussion by MisherLiu** [UI5/webcomponents#3733] [Q&A]
Hi Dev Team,

I write a demo using UI5 Web Component in Vue.js, while the ui5 webcomponent size of the static assest  is about 128KB, not small like mentioned in the home of this github. Refer screenshot:
![image](https://user-images.githubusercontent.com/64764287/130920928-bda7fbb2-e77b-4439-9cb4-c0f2511b3651.png)
 Or you can refer [link here](https://pages.github.tools.sap/ibso-ui-guideline-reference/vue_demo/#/) to run the Lighthouse report in Chrome devTools and see the result in Diagnostics

Could you guide on this?

**Comment by vladitasev** [comment]
Hi @MisherLiu 

We'll rephrase: "A rich set of enterprise-grade reusable UI elements driven by a lightweight framework (~20K gzipped)." so that it's clear that the "20kb gzipped" part only refers to the framework codebase. The more components one bundles next to the framework, the more size will increase. 

Regards,
Vladi
